src/test/java/org/takes/facets/auth/PsBasicTest.java

```diff
@@ -209,4 +211,63 @@ private String generateAuthenticateHead(
         );
         return String.format(AUTH_BASIC, encoded);
     }
+
+    /**
+     * Test of {@link PsBasic.Default}.
+     * @author lcozzani (lautaromail@gmail.com)
+     * @version $Id$
+     * @since 0.22
+     */
+    public static final class Default {
+
+        /**
+         * Test {@link org.takes.facets.auth.PsBasic.Default#enter}.
```

Comment:
@lautarobock Comment should say something like:

> PsBasic.Default can return a valid user.

See [this post](http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html?expand=1#test-method-names) for a guideline.